rpcs3/Emu/RSX/RSXFIFO.cpp

```diff
@@ -424,9 +415,8 @@ namespace rsx
 	{
 		FIFO::register_pair command;
 		fifo_ctrl->read(command);
-		const auto cmd = command.reg;
 
-		if (UNLIKELY(cmd & (0xffff0000 | RSX_METHOD_NON_METHOD_CMD_MASK)))
+		if (const u32 cmd = command.reg; UNLIKELY(cmd & RSX_METHOD_NON_METHOD_CMD_MASK))
```

Comment:
unnecessary stylistic change. It would have been ok for new code, but when changing existing it just increases the size of the diff with no gain. Its a temp in both cases.